charts/argo-cd/values.yaml

```diff
@@ -2217,6 +2217,24 @@ repoServer:
   #  - name: cmp-tmp
   #    emptyDir: {}
 
+  # -- Volumes to be used in replacement of emptydir on default volumes
+  existingVolumes: {}
+  #  gpg_keyring:
```

Comment:
@jmeridth I'm generally OK with having this as opt-in. Not sure if we want to have mixed syntax here. We use `camelCase` everywhere but volume suddenly changes to `lower_underscore`.

Reply:
Looks good and a good shout about naming.

Reply:
@pdrastil @mbevc1  volumes name are lower_underscore to keep the very same name as volumeMount names into chat but no problem to change them camelCase if you wish them to match global naming. 
Please just confirm if you want me to change this.

Reply:
I don't have a strong preference and we should aim for consistency.

Reply:
moved to camelCase